packages/teams-js/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@microsoft/teams-js",
   "author": "Microsoft Teams",
-  "version": "2.3.0",
+  "version": "2.5.0",
```

Comment:
Please don't manually change the version of the Teams JS package

Reply:
Sorry that was so I can test locally... It's reverted now